lib/modules/dosomething/dosomething_northstar/dosomething_northstar.auth.inc

```diff
@@ -173,15 +173,32 @@ function dosomething_northstar_perform_authorization_actions() {
  * @param $account
  */
 function dosomething_northstar_user_logout($account) {
-  $is_openid_login = ! empty($_SESSION['DOSOMETHING_NORTHSTAR_OPENID_CONNECT']);
-  $northstar_logout_url = url(NORTHSTAR_URL . '/logout', [
-    'query' => ['redirect' => url('/', ['absolute' => TRUE])],
-  ]);
+  $user = user_load($account->uid);
 
   // Destroy the current session, and reset $user to the anonymous user.
   session_destroy();
 
-  drupal_goto($is_openid_login ? $northstar_logout_url : '/');
+  $is_openid_login = ! empty($user->field_refresh_token[LANGUAGE_NONE][0]['value']);
```

Comment:
I'm fairly certain it will always be saved under `LANGUAGE_NONE` because that's [how the `dosomething_user_set_fields` function saves it](https://github.com/DoSomething/phoenix/blob/085eed871725ce89425bf19ff99515f3d3ef1c76/lib/modules/dosomething/dosomething_user/dosomething_user.module#L1383), but I'm not sure if there's anything else that could hook in and cause trouble. @angaither?